control/rlocus.py

```diff
@@ -93,52 +102,164 @@ def root_locus(sys, kvect=None, xlim=None, ylim=None, plotstr='-', Plot=True,
     (nump, denp) = _systopoly1d(sys)
 
     if kvect is None:
-        kvect = _default_gains(sys)
-
-    # Compute out the loci
-    mymat = _RLFindRoots(sys, kvect)
-    mymat = _RLSortRoots(sys, mymat)
-
-    # Create the plot
-    if (Plot):
+        gvect, mymat, xl, yl = _default_gains(nump, denp, xlim, ylim)
+    else:
+        gvect = np.asarray(kvect)
+        mymat = _find_roots(nump, denp, gvect)
+        mymat = _sort_roots(mymat)
+        xl = _ax_lim(mymat)
+        yl = _ax_lim(mymat * 1j)
+
+    # Create the Plot
+    if Plot:
         f = pylab.figure()
         if PrintGain:
             f.canvas.mpl_connect(
-                'button_release_event', partial(_RLFeedbackClicks, sys=sys))
+                'button_release_event', partial(_feedback_clicks, sys=sys))
         ax = pylab.axes()
 
-        # plot open loop poles
+        # Plot open loop poles
```

Comment:
I might be missing something, but it is not clear why you changed these "plot" words to uppercase "Plot". Is it merely a style change? If so, can you place it in a separate commit and indicate that it is a trivial change and only affects comments?